server/sonar-web/src/main/js/apps/coding-rules/workspace-list-item-view.js

```diff
@@ -69,15 +69,15 @@ export default WorkspaceListItemView.extend(RuleFilterMixin).extend({
   activate () {
     const that = this;
     const selectedProfile = this.options.app.state.get('query').qprofile;
-    const othersQualityProfiles = _.reject(this.options.app.qualityProfiles, function (profile) {
+    const othersQualityProfiles = _.reject(this.options.app.qualityProfiles, profile => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)